stories/components/theme-toggle/themeToggle.js

```diff
@@ -58,21 +58,39 @@ body {
 `;
 
 export const themingWithoutToggle = () => html`
+  <h2>Style the background of a component</h2>
+  <p>The login components are in the default light theme on a light
+  background. We are setting the background of the component with id "login-two"
+  to the <code>aquamarine</code> color using CSS. Check the CSS tab.</p>
```

Comment:
can we please show the use of the `--fill-color` variable either in this story or another story to illustrate how a developer can use the background color from the fluent design system?